mcs/class/corlib/Makefile

```diff
@@ -203,9 +203,12 @@ TEST_MONO_PATH=$(PWD)/../lib/$(PROFILE)
 
 run-test-vts: test-vts
 	@echo Running vts tests...
-	PATH="$(TEST_RUNTIME_WRAPPERS_PATH):$(PATH)" $(TEST_RUNTIME) $(RUNTIME_FLAGS) $(TEST_HARNESS) $(NOSHADOW_FLAG) \
+	PATH="$(TEST_RUNTIME_WRAPPERS_PATH):$(PATH)" $(TEST_RUNTIME) $(TEST_RUNTIME_FLAGS) $(TEST_HARNESS) $(NOSHADOW_FLAG) \
 		$(vtsdir)/$(PROFILE)_TestLib/BinarySerializationOverVersions.exe
 
+run-test-interp:
+	$(MAKE) run-test TEST_RUNTIME_FLAGS='--interpreter' TEST_HARNESS_EXCLUDES='$(TEST_HARNESS_EXCLUDES_INTERP)'
```

Comment:
instead of needing to add such a target to every assembly we enable on the interpreter I think it'd be cleaner if we had a variable e.g. `TEST_WITH_INTERPRETER` that we can set from the outside and is checked in tests.make, so setting  TEST_RUNTIME_FLAGS and TEST_HARNESS_EXCLUDES only happens there.

I'm fine if this cleanup is done in a follow up PR.

Reply:
sounds good!

Reply:
@akoeplinger I've updated this PR regarding your suggestion.  It would be at least already useful for another test suite (soft debugger tests, https://github.com/mono/mono/pull/5943 ).